packages/lesswrong/lib/helpers.ts

```diff
@@ -191,3 +191,9 @@ export const setAtPath = <T extends {}, V extends AnyBecauseHard>(
   }
   return value;
 }
+
+/*
+* @deprecated Only use for debugging, or else litter the console with logs
+*/
+// eslint-disable-next-line no-console
+export const trace = <A>(msg: string) => (a: A): A => { console.log(msg, a); return a }
```

Comment:
@rjmk this & uses of it should probably be replaced by `loggerConstructor > logger` (sorry, am aware that is probably more annoying to pipe through `then`s).